packages/element-templates-json-schema/test/fixtures/invalid-keywords.js

```diff
@@ -0,0 +1,39 @@
+export const template = {
+  'name': 'Keywords',
+  'id': 'com.camunda.example.Keywords',
+  'appliesTo': [
+    'bpmn:Task'
+  ],
+  'properties': [],
+  keywords: 123
```

Comment:
Keep existing coding style.

```suggestion
  'keywords': 123
```

Reply:
> Keep existing coding style.

I've never considered this to be code style. For me, it was just an outcome of copying the JSON to JS :D

Reply:
This is bike-shedding :wink: 

Point is: We should be consistent with the existing code-base. This is the reason we don't do

```
var foo = "a" + 'b' + `c`.